docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
```

Comment:
Not opposed to a different paradigm but live version should be 1

Reply:
Oh this `version: '3'` sets the Docker Compose version rather than the app version. Here's [more reference](https://docs.docker.com/compose/compose-file/) if you want.